### ISSUE TITLE
[Mac] Could not connect to any provider in 2.5.1

### ISSUE THREAD

**Issue by etherunit** [mysteriumnetwork/node#2543]
**Describe the bug**
Could not connect to any selected provider. Each connection attempt gives an error.

**To Reproduce**
1. Select any provider
2. Click Connect

**Expected behavior**
Consumer can connect to the selected node.

**Screenshots**
<img width="687" alt="Screenshot 2020-07-28 at 16 55 43" src="https://user-images.githubusercontent.com/60128746/88675504-add0e400-d0f3-11ea-8f2c-5dbedc8b6933.png">
<img width="696" alt="Screenshot 2020-07-28 at 16 56 07" src="https://user-images.githubusercontent.com/60128746/88675532-b1fd0180-d0f3-11ea-97ca-a3b4d7567932.png">

**Environment (please complete the following information):**
 - Node version: Any
 - OS: mac version 10.15.5 (19F101)

**Additional context**

supervisor.log

```
2020-07-27T18:00:58.000 ??? DEBUG: (utun6) 2020/07/27 18:00:58 Routine: encryption worker - stopped
2020-07-27T18:00:58.000 ??? DEBUG: (utun6) 2020/07/27 18:00:58 Routine: handshake worker - stopped
2020-07-27T18:00:58.000 ??? DEBUG: (utun6) 2020/07/27 18:00:58 Routine: handshake worker - stopped
2020-07-27T18:00:58.000 ??? DEBUG: (utun6) 2020/07/27 18:00:58 Routine: decryption worker - stopped
2020-07-27T18:00:58.000 ??? DEBUG: (utun6) 2020/07/27 18:00:58 Routine: decryption worker - stopped
2020-07-27T18:00:58.000 ??? DEBUG: (utun6) 2020/07/27 18:00:58 peer(h3z8…jBhM) - Routine: sequential receiver - stopped
2020-07-27T18:00:58.000 ??? DEBUG: (utun6) 2020/07/27 18:00:58 peer(h3z8…jBhM) - Routine: nonce worker - stopped
2020-07-27T18:00:58.000 ??? DEBUG: (utun6) 2020/07/27 18:00:58 peer(h3z8…jBhM) - Routine: sequential sender - stopped
2020-07-27T18:00:58.000 ??? INFO: (utun6) 2020/07/27 18:00:58 Interface closed
2020-07-27T18:00:58.000 ERR Could not clean DNS error="DNS is already cleaned"
2020-07-27T18:00:58.000 ERR wg-up failed error="could not setup network: failed to assign IP address: "ifconfig utun6 10.182.0.2/24 10.182.0.1": signal: killed output: : signal: killed"
2020-07-27T18:01:57.000 INF Trying to stop node process gracefully
2020-07-27T18:01:58.000 ERR Could not send message: "ok" error="write unix /var/run/myst.sock->: write: broken pipe"
thomastsipoulanis@Thomac ~ %
```
[main (5).log](https://github.com/mysteriumnetwork/node/files/4988817/main.5.log)
[renderer (2).log](https://github.com/mysteriumnetwork/node/files/4988818/renderer.2.log)
[message (3).txt](https://github.com/mysteriumnetwork/node/files/4988826/message.3.txt)

**Comment by hubthom** [comment]
an hour  ago  i   have  installed  the MacOS version 10.15.6 (19G73), and  now the issue it is solved.

**Comment by xapobilo** [comment]
I've got the same problem on Windows 10 with the same Version of Mysterium. 

Several times tried to topup but it didn't work for me. finally without no reason tokens appeared in the app wallet.

After 3rd attempt of connecting to one provider the app tries to disconnect, but finally i have to enforce the app to get closed. 

After reopening the app the balance is 0 MYSTT.

[mysterium-node.log](https://github.com/mysteriumnetwork/node/files/5005769/mysterium-node.log)
[myst_supervisor.log](https://github.com/mysteriumnetwork/node/files/5005771/myst_supervisor.log)

**Comment by vkuznecovas** [comment]
This is solved for 0.39
